law_reader/common/RevisionSummaryInfo.py

```diff
@@ -0,0 +1,13 @@
+class RevisionSummaryInfo:
+    """
+    Contains information relevant to a revision summary in the Supabase database
+    """
+    def __init__(self, revision_guid: str, rt_unique_id: int, revision_internal_id: int):
+        self.revision_guid = revision_guid
+        self.rt_unique_id = rt_unique_id
+        self.revision_internal_id = revision_internal_id
+        self.full_text = None
```

Comment:
I removed these as attributes of the object, since they're just called the one time within the for loop, so that removes the issue of this being a mutable object.